src/resources/views/crud/fields/radio.blade.php

```diff
@@ -2,6 +2,12 @@
 @php
     $optionValue = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? '';
```

Comment:
Thanks guys!

I'm wondering - wouldn't a quick cast to integer do the same, without going through all the trouble of using Arr and going to the model? If the problem is that, when cast to boolean, the $optionValue would be false/true, not 0/1, we could just do this, couldn't we?

```suggestion
    $optionValue = (int)old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? '';
```

Reply:
I don't think so. What would happen to: 

```php
CRUD::field('some')->type('radio')->options(['hey' => 'Hey', 'sure' => 'Sure']);

(int)'hey'; // result 0.
```

Reply:
Oups. Right! 😀